apps/tunes/serializers.py

```diff
@@ -22,6 +22,12 @@ class Meta:
         fields = ('artist', 'name', 'genre', 'code')
 
 
+class LastPlaylistSerializer(serializers.Serializer):
```

Comment:
We need to add the `description` field to this serializer, in case a user had set it when generating the browse playlist